src/util.cpp

```diff
@@ -7,6 +7,11 @@
 #include "config/bitcoin-config.h"
 #endif
 
+#if (defined(__FreeBSD__) || defined(__OpenBSD__))
```

Comment:
I suppose you're right, however there is namespace pollution in the bitcoin includes, so if I move it too much down stuff breaks dramatically in the openbsd sys/* includes.